spec/opera/operation/base_spec.rb

```diff
@@ -559,6 +559,31 @@ def step_2
         end
       end
 
+      context 'for accessors/readers' do
+        subject { operation_class.call(params: {}, dependencies: {}) }
+
+        let(:operation_class) do
+          Class.new(Operation::Base) do
+            params_reader :bar, default: -> { :bar }
```

Comment:
@petergebala I think this spec isn't covering this fix properly. I was trying to check it locally and if the param has a default the issue is not happening. Do you mind adding another param reader or another context to cover this?

Reply:
@BravoSimone was right. I missed one change in specs. Fixed and pushed with force